client/state/selectors/get-original-user-setting.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Given a settingName, returns that original setting if it exists or null
+ *
+ * @param  {Object} state Global state tree
+ * @param  {String} settingName - setting name
+ * @return {*} setting key value
+ */
+export default function getOriginalUserSetting( state, settingName ) {
+	return get( state.userSettings.settings, [ settingName ], null );
```

Comment:
seems a bit odd to use the safe-defaulting getter and then doing the nested property access in the _unsafe_ part of the function call…

```js
get( state, [ 'userSettings', 'settings', settingName ], null );
```